README.md

```diff
@@ -1,2 +1,7 @@
 # terraform-aws-cis-fundamentals
+
 This Terraform module helps to setup an AWS account with the requirements of  CIS Amazon Web Services Foundations Benchmark v1.1.0
+
+1. Identity and Access Management
+    1. Avoid the use of the "root" account (Scored) - Cannot be cofidied
+    2. Ensure multi-factor authentication (MFA) is enabled for all IAM users that have a console password (Scored)
```

Comment:
Consider _all IAM users with console access_ as opposed to _all IAM users that have a console password_.

Reply:
I haven't changed this due to I've copied this from CIS documentation without changing a single word. I'm planning to keep like this as it will be way much easier to update when the new version of the benchmark is released.